frontend/public/components/utils/log-window.jsx

```diff
@@ -151,7 +153,12 @@ export const LogWindow = withTranslation()(LogWindowWithTranslation);
 LogWindow.propTypes = {
   bufferFull: PropTypes.bool.isRequired,
   lines: PropTypes.array.isRequired,
+  wrapLines: PropTypes.bool,
   linesBehind: PropTypes.number.isRequired,
   status: PropTypes.string.isRequired,
   updateStatus: PropTypes.func.isRequired,
 };
+
+LogWindow.defaultProps = {
+  wrapLines: false,
```

Comment:
On the fence about whether this is necessary.  @spadgett, what's your opinion?

Reply:
We've largely switched to functional components and hooks where we use es6 default values. It's probably a good practice for class components... I'd keep it.